python/kwiver/sprokit/adapters/adapter_data_set.cxx

```diff
@@ -171,9 +203,14 @@ PYBIND11_MODULE(adapter_data_set, m)
 
     .def("add_datum", &ka::adapter_data_set::add_datum)
 
-    // General get_value which gets data of any type from a port and __getitem__
+    // General methods for retrieving data with an arbitrary type from a port.
     .def("get_port_data", &kwiver::sprokit::python::get_port_data_correct_type,
           "This method is equivalent to using __getitem__")
+    .def("value", &kwiver::sprokit::python::get_port_data_correct_type,
+          "This method is equivalent to using __getitem__")
+    .def("value_or", &kwiver::sprokit::python::value_or_correct_type,
+          py::arg("port"), py::arg("value_if_missing") = py::none(),
+          "This method is equivalent to using __getitem__")
```

Comment:
Should this be changed to `equivalent of get method` since there is  default value option https://www.tutorialspoint.com/python/dictionary_get.htm

Reply:
Yes, good call

Reply:
See 5fa1a91415b30728b9eee0a48b2ca8156e75368f